app/assets/javascripts/toggle_name_and_email_input.js

```diff
@@ -7,16 +7,20 @@
         var link = $($(this).data('link'));
         var targetScope = $(this).data('target');
         var store = target.detach();
-        var form = $('form').last();
+        var contentForm = $('form:last');
```

Comment:
This is absolutely leaking outside of the context of this plugin.  The jQuery selector will select anything on the page that it is called (regardless of how the plugin was instantiated).  That's just how jQuery works.

I'm not sure why `closest()` didn't work however that doesn't mean we should allow our plugin's behavior to leak outside of our scope.